packages/wallet-adapter-core/package.json

```diff
@@ -44,13 +44,14 @@
     "tsup": "^5.10.1",
     "typescript": "^4.5.3"
   },
+  "peerDependencies": {
+    "@aptos-labs/ts-sdk": "^1.10.0"
+  },
```

Comment:
was moving peer deps to the top intentional?

Reply:
no... just formatting I guess..